packages/components/molecules/f-skeleton-loader/src/components/SkeletonLoader.vue

```diff
@@ -2,13 +2,9 @@
 <template>
     <div
         data-test-id="skeletonLoader"
-        :class="[
-            'c-skeleton-loader',
-            {
-                ['c-skeleton-loader--animated']: isAnimated
-            }
-        ]">
-        <component :is="skeletonType" />
+        :class="$style['c-skeleton-loader']">
```

Comment:
Should be `c-skeletonLoader` to match naming conventions

Reply:
This was removed in 0.4.0 as there was no requirement to actually support it.